src/univariate/continuous/rayleigh.jl

```diff
@@ -79,7 +79,7 @@ end
 
 function logccdf(d::Rayleigh, x::Real)
     z = - x^2 / (2 * d.σ^2)
-    return x > 0 ? z : isnan(x) ? oftype(z, NaN) : zero(x)
+    return x >= 0 ? z : isnan(x) ? oftype(z, NaN) : -zero(z)
```

Comment:
Yeah, this is simpler (thought about it actually but assumed that it doesn't matter if we return negative or positive zeros for `logccdf`). Could you add the comment about positive cdf here? Otherwise I assume it can be a bit unclear for a reader why `-zero(z)` is used here.